flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/source/reader/fetcher/SingleThreadFetcherManager.java

```diff
@@ -93,6 +93,14 @@ public void addSplits(List<SplitT> splitsToAdd) {
         }
     }
 
+    @Override
+    public void finishSplits(List<SplitT> finishedSplits) {
```

Comment:
Would it be better to name it as `removeSplits()` for consistency with `addSplits()`?

Reply:
I will fix it.